### PR TITLE
docs: how to add an insecure registry

### PR DESCRIPTION
**Pull request by cdrage**
docs: how to add an insecure registry

### What does this PR do?

Adds documentation on how to add an insecure registry to Podman Desktop
which consists of adding it via the Registries section as well as
modifying the Podman VM / podman `registries.conf` file

### Screenshot/screencast of this PR

See site

### What issues does this PR fix or reference?

Closes last part of https://github.com/containers/podman-desktop/issues/1653

### How to test this PR?

1. Preview the site using `yarn website:dev` for any errors
2. Go through the steps.

### CODE REVIEW

File: website/docs/getting-started/insecure-registry.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+tags: [pods]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Adding an Insecure Registry to Podman Desktop
+
+In this guide, you will learn how to add an insecure registry to Podman Desktop. An insecure registry allows you to pull and push container images over an unencrypted HTTP connection instead of a secure HTTPS connection. Please note that using an insecure registry can expose your data to security risks, so it's recommended to use secure connections whenever possible.
+
```
Comment (benoitf) [inline website/docs/getting-started/insecure-registry.md]: I think insecure is not only unencrypted HTTP connection but also just a self-signed certificate (so we don't trust the certificate)

**Review by benoitf — COMMENTED** [review]
I think the image 
image

is not helpful, I think I would expect to see the full screen and then an highlight on the button that I need to click on

I see Tabs is imported in the .md but it's not used ?

I think it'll really helpful to have a per OS section or a Windows/mac tab and a Linux tab as steps are 'with a machine' and some are 'without a machine'

**Comment by cdrage** [comment]
Updated! Thanks for the review / I pushed the correct branch and updated the first introductory paragraph.

**Review by benoitf — APPROVED** [review]

File: website/docs/getting-started/insecure-registry.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+<Tabs groupId="operating-systems">
+<TabItem value="macwin" label="macOS & Windows">
+
+1. Add your insecure registry within Settings -> Registries.
+
+2. Click "Yes" to the insecure registry warning.
+
+![Podman Desktop Registry Warning](img/registry-warning.png)
+
+3. SSH into the Podman Machine in order to edit `registries.conf`.
```
Comment (benoitf) [inline website/docs/getting-started/insecure-registry.md]: looks like there is a vale error there
```suggestion
3. SSH into the Podman Machine to edit `registries.conf`.
```

File: website/docs/getting-started/insecure-registry.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+#### Prerequisites
+
+- Podman installed and running (VM if on macOS or Windows).
+
+#### Procedure
+
+<Tabs groupId="operating-systems">
+<TabItem value="macwin" label="macOS & Windows">
+
+1. Add your insecure registry within Settings -> Registries.
```
Comment (benoitf) [inline website/docs/getting-started/insecure-registry.md]: I think @themr0c is using `**<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**`
**<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**

File: website/docs/getting-started/insecure-registry.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+2. Click "Yes" to the insecure registry warning.
+
+![Podman Desktop Registry Warning](img/registry-warning.png)
+
+3. SSH into the Podman Machine in order to edit `registries.conf`.
+
+```sh
+$ podman machine ssh
```
Comment (benoitf) [inline website/docs/getting-started/insecure-registry.md]: should we say `podman machine ssh [optional-machine-name]`

File: website/docs/getting-started/insecure-registry.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+$ podman machine stop
+$ podman machine start
+```
+
+8. Pull or push container images: You can now use Podman Desktop to pull or push container images from/to the insecure registry you added.
+
+</TabItem>
+<TabItem value="linux" label="Linux">
+
+1. Add your insecure registry within Settings -> Registries.
```
Comment (benoitf) [inline website/docs/getting-started/insecure-registry.md]:
```suggestion
1. Add your insecure registry within **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**.
```

File: website/docs/getting-started/insecure-registry.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+1. Click "Yes" to the insecure registry warning.
+
+![Podman Desktop Registry Warning](img/registry-warning.png)
```
Comment (benoitf) [inline website/docs/getting-started/insecure-registry.md]: maybe image name should describe more the issue like registry-warning-insecure.png (so it won't conflict with others images we may add later)

**Comment by cdrage** [comment]
Thanks for the approval! I've updated all the suggestions and will mark this as auto-merge

**Pull request merged**